docs/Tutorial_STUMPY_Basics.ipynb

```diff
@@ -392,6 +392,25 @@
     "mp[motif_idx, 0]"
```

Comment:
<p>A minor typo in the inline comment provided in the code cell.&nbsp;</p>

<pre class="ql-syntax" spellcheck="false">Replace matric with matrix
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/972/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>